components/layout_thread/lib.rs

```diff
@@ -1333,7 +1354,14 @@ impl LayoutThread {
             page_clip_rect: max_rect(),
         };
 
-        let mut layout_context = self.build_layout_context(&*rw_data, false);
+        // Clone an Arc so that we don’t keep borrowing `self`.
```

Comment:
Ah yes, this was for per-document locks that I ended up giving up on. A removed that Arc and clone before committing but forgot to remove the comment. Fixed.